### ISSUE TITLE
Feature Request: Update Diff Count as Kill Counts Progress

### ISSUE THREAD

**Issue by CapWacky** [WarmedxMints/ODMissionStacker#8]
Not sure if this is the best place to put a feature request, feel free to redirect me if there is a better place for such things:

It would be nice to have the Diff Count in the lower right section for each faction count down as your kills progress. This would help if you do turn ins and picking up more missions before your current set is complete. Thus you can see the updated Diff count for which factions to pick up missions from vs not.

At any rate thanks for making a great tool, your OD Explorer is also excellent.

**Comment by WarmedxMints** [comment]
Unfortunately, the kill count is very hit and miss as FDEV does not provide a mission updated event so the best that can be done is to count the kills as a bounty event occurs and then calculate which mission to assign them to.  This is error-prone though, especially when in a wing.

So what I do instead is exclude any missions which have had the redirected event fired from that information as we know the kills are completed on them.

**Comment by ternite** [comment]
It took me some time to even understand what the lower right table is showing. Only after summing some table values in my head, I understood, that the kill count placed there is the sum of all kills needed for each faction, but only for missions still incomplete.

I was expecting an overview of how many kills will still be needed to complete that faction's missions.

I came here to place the same Feature Request as OP. After a kill, the tool updates the upper kill count, why not removing a (still needed) kill from the lower right table, too? That way, the information would be more useful, as you can see how many kills will be needed to empty the whole stack. That it's not 100% accurate can't be the reason to not update the value below - because that would also be the same reason to not update the kill counts in the missions list.

(I even tried to do the change myself in Visual Studio, and maybe offering a pull request, but dependency horror creeped into my VS and let me finallly abstain from this endeavor.)
